.nvmrc

```diff
@@ -1 +1 @@
-18.14.1
+18.18.2
```

Comment:
This update is necessary because I switched to using native `fetch` and some bugfixes happened on it between v18.14.1 and v18.18.2.